### PR TITLE
api modification

### PR DESCRIPTION
**Pull request by zy0531**
half modification of fluid/io/layers according to 1.3->develop Report

### CODE REVIEW

File: doc/fluid/api_cn/fluid_cn.rst [hunk 1]
```diff
@@ -188,18 +188,25 @@ str类型。在 ``ParallelExecutor`` 中，存在两种减少策略（reduce str
 
 .. py:attribute:: remove_unnecessary_lock
 
-BOOL类型。如果设置为True, GPU操作中的一些锁将被释放，ParallelExecutor将运行得更快，默认为 False。
+BOOL类型。如果设置为True, GPU操作中的一些锁将被释放，ParallelExecutor将运行得更快，默认为 True。
 
+.. py:attribute:: sync_batch_norm
+
+类型为bool，sync_batch_norm表示是否使用同步批处理规范化，在训练阶段通过多个设备同步均值和方差。
```
Comment (haowang101779990) [inline doc/fluid/api_cn/fluid_cn.rst]:
```suggestion
类型为bool，sync_batch_norm表示是否使用同步的batch正则化，即在训练阶段通过多个设备同步均值和方差。
```

File: doc/fluid/api_cn/fluid_cn.rst [hunk 2]
```diff
@@ -188,18 +188,25 @@ str类型。在 ``ParallelExecutor`` 中，存在两种减少策略（reduce str
...
 .. py:attribute:: remove_unnecessary_lock
 
-BOOL类型。如果设置为True, GPU操作中的一些锁将被释放，ParallelExecutor将运行得更快，默认为 False。
+BOOL类型。如果设置为True, GPU操作中的一些锁将被释放，ParallelExecutor将运行得更快，默认为 True。
 
+.. py:attribute:: sync_batch_norm
+
+类型为bool，sync_batch_norm表示是否使用同步批处理规范化，在训练阶段通过多个设备同步均值和方差。
+
+当前的实现不支持FP16培训和CPU。只有一台机器是同步的，不是所有的机器。
```
Comment (haowang101779990) [inline doc/fluid/api_cn/fluid_cn.rst]:
```suggestion
当前的实现不支持FP16培训和CPU。仅在一台机器上进行同步式批正则，不适用于多台机器。
```

File: doc/fluid/api_cn/fluid_cn.rst [hunk 3]
```diff
@@ -969,13 +977,12 @@ program中所有的算子会按顺序执行。
...
-提示：你可以用 ``Executor`` 来调试基于并行GPU实现的复杂网络，他们有完全一样的参数也会产生相同的结果。
-
-
 .. py:method:: close()
 
 
-关闭这个执行器(Executor)。调用这个方法后不可以再使用这个执行器。 对于分布式训练, 该函数会释放在PServers上涉及到目前训练器的资源。
+关闭这个执行器(Executor)。
+
+调用这个方法后不可以再使用这个执行器。 对于分布式训练, 该函数会释放在PServers上涉及到目前训练器的资源。
```
Comment (haowang101779990) [inline doc/fluid/api_cn/fluid_cn.rst]:
```suggestion
调用这个方法后不可以再使用这个执行器。 对于分布式训练, 该函数会释放在PServers上和目前Trainer有关联的资源。
```

File: doc/fluid/api_cn/fluid_cn.rst [hunk 4]
```diff
@@ -1000,12 +1007,12 @@ feed map为该program提供输入数据。fetch_list提供program训练结束后
...
 	- **program** (Program|CompiledProgram) – 需要执行的program,如果没有给定那么默认使用default_main_program (未编译的)
 	- **feed** (dict) – 前向输入的变量，数据,词典dict类型, 例如 {“image”: ImageData, “label”: LabelData}
-	- **fetch_list** (list) – 用户想得到的变量或者命名的列表, run会根据这个列表给与结果
+	- **fetch_list** (list) – 用户想得到的变量或者命名的列表, 该方法会根据这个列表给出结果
 	- **feed_var_name** (str) – 前向算子(feed operator)变量的名称
 	- **fetch_var_name** (str) – 结果获取算子(fetch operator)的输出变量名称
 	- **scope** (Scope) – 执行这个program的域，用户可以指定不同的域。缺省为全局域
 	- **return_numpy** (bool) – 如果为True,则将结果张量（fetched tensor）转化为numpy
-	- **use_program_cache** (bool) – 当program较上次比没有改动则将其置为True
+	- **use_program_cache** (bool) – 是否跨批使用缓存程序设置。只有当（1）程序没有用数据并行编译，并且（2）program、 feed变量名和fetch_list变量名与上一步相比没有更改时，设置为True的速度才会更快。
```
Comment (haowang101779990) [inline doc/fluid/api_cn/fluid_cn.rst]:
```suggestion
	- **use_program_cache** (bool) – 是否跨批使用缓存程序设置。设置为True时，只有当（1）程序没有用数据并行编译，并且（2）program、 feed变量名和fetch_list变量名与上一步相比没有更改时，运行速度才会更快。
```

File: doc/fluid/api_cn/layers_cn.rst [hunk 5]
```diff
@@ -1495,6 +1495,7 @@ affine_channel
 	- **bias** (Variable):形状为(C)的一维输入，第C个元素是输入的第C个通道的仿射变换的偏置。
 	- **data_layout** (string, default NCHW): NCHW 或 NHWC，如果输入是一个2D张量，可以忽略该参数
 	- **name** (str, default None): 此层的名称
+    - **act** (str, default None): 用于该层的输出的激活
```
Comment (haowang101779990) [inline doc/fluid/api_cn/layers_cn.rst]:
```suggestion
        - **act** (str, default None): 应用于该层输出的激活函数
```

File: doc/fluid/api_cn/layers_cn.rst [hunk 6]
```diff
@@ -1680,11 +1681,11 @@ batch_norm
...
 
 参数：
-    - **input** (Variable) - 输入变量，为LoDTensor
+    - **input** (Variable) - 输入变量的排序，可以为 2, 3, 4, 5
     - **act** （string，默认None）- 激活函数类型，linear|relu|prelu|...
-    - **is_test** （bool,默认False） - 标志位，是否用于测试或训练
-    - **momentum** （float，默认0.9）- （暂无说明，待更新）
-    - **epsilon** （float，默认1e-05）- （暂无说明，待更新）
+    - **is_test** （bool,默认False） - 指示它是否在测试阶段。
+    - **momentum** （float，默认0.9）- 此值用于计算 moving_mean and moving_var. 更新公式为: \(moving\_mean = moving\_mean * momentum + new\_mean * (1. - momentum)\) \(moving\_var = moving\_var * momentum + new\_var * (1. - momentum)\)， 默认值0.9.
```
Comment (haowang101779990) [inline doc/fluid/api_cn/layers_cn.rst]:
```suggestion
    - **momentum** （float，默认0.9）- 此值用于计算 moving_mean and moving_var. 更新公式为:  :math:`\(moving\_mean = moving\_mean * momentum + new\_mean * (1. - momentum)\)` :math:`\(moving\_var = moving\_var * momentum + new\_var * (1. - momentum)\)` ， 默认值0.9.
```

File: doc/fluid/api_cn/layers_cn.rst [hunk 7]
```diff
@@ -3179,7 +3179,16 @@ LSTMP层(具有循环映射的LSTM)在LSTM层后有一个分离的映射层，
     - **candidate_activation** (str) - 候选隐藏状态（candidate hidden state）的激活状态。Choices = [“sigmoid”，“tanh”，“relu”，“identity”]，默认“tanh”。
     - **proj_activation** (str) - 投影输出的激活函数。Choices = [“sigmoid”，“tanh”，“relu”，“identity”]，默认“tanh”。
     - **dtype** (str) - 数据类型。Choices = [“float32”，“float64”]，默认“float32”。
-    - **name** (str|None) - 该层名称（可选）。若设为None，则自动为该层命名。 
+    - **name** (str|None) - 该层名称（可选）。若设为None，则自动为该层命名。
+    - **h_0** (Variable) - 初始隐藏状态是可选输入，默认为0。这是一个具有形状的张量(N x D)，其中N是批大小，D是投影大小。 
+    - **c_0** (Variable) - 初始单元格状态是可选输入，默认为0。这是一个具有形状(N x D)的张量，其中N是批大小。h_0和c_0可以为空，但只能同时为空。
```
Comment (haowang101779990) [inline doc/fluid/api_cn/layers_cn.rst]:
```suggestion
    - **c_0** (Variable) - 初始cell状态是可选输入，默认为0。这是一个具有形状(N x D)的张量，其中N是批大小。h_0和c_0可以为空，但只能同时为空。
```

**Review by haowang101779990 — COMMENTED** [review]
Textual Fix

File: doc/fluid/api_cn/fluid_cn.rst [hunk 8]
```diff
@@ -192,9 +192,9 @@ BOOL类型。如果设置为True, GPU操作中的一些锁将被释放，Paralle
 
 .. py:attribute:: sync_batch_norm
 
-类型为bool，sync_batch_norm表示是否使用同步批处理规范化，在训练阶段通过多个设备同步均值和方差。
+类型为bool，sync_batch_norm表示是否使用同步的batch正则化，即在训练阶段通过多个设备同步均值和方差。
 
-当前的实现不支持FP16培训和CPU。只有一台机器是同步的，不是所有的机器。
+当前的实现不支持FP16培训和CPU。仅在一台机器上进行同步式批正则，不适用于多台机器。
```
Comment (shanyi15) [inline doc/fluid/api_cn/fluid_cn.rst]: image
是batch正则还是批正则呢，请统一说法

**Comment by shanyi15** [comment]
存在一个conflict，请修复

**Review by shanyi15 — APPROVED** [review]

**Pull request merged**